### DISCUSSION TITLE
how to start this project

### DISCUSSION THREAD

**Discussion by anujkesharwani** [wagtail/gsoc#5] [General]
please brief some user interface and requirement of backend  collection .
i have learn **python** in advance.

**Comment by chamoli2k2** [comment]
@thibaudcolas will brief us all about project detail soon, until then look existing code base and also try to learn technology described in the tech stack of project :)

**Reply by anujkesharwani** [reply]
thanks

**Comment by anujkesharwani** [comment]
sir, please tell me REDORANGEONE  have many repo.
which is related GSOC 2023.
please , brief  any project or provide me documentation any gsoc project.
I am really very excited this is my first open source project contribution .

**Reply by sahil-sagwekar2652** [reply]
Hello @anujkesharwani,

You can go through the documentation for setting up and contributing to Wagtail [here (https://docs.wagtail.org/en/stable/contributing/index.html). 

Use the '@' symbol to tag user accounts.

**Reply by anujkesharwani** [reply]
thanks sir

**Comment by thibaudcolas** [comment]
Hi @anujkesharwani, I’m afraid your question isn’t very clear as-is. Please have a read through our [contributor guide](https://github.com/wagtail/gsoc/blob/main/contributor-guide.md). GSoC projects don’t start until candidates have put in an application and been accepted. The whole timeline is very well explained on Google’s contributor guide (linked from ours).
